src/Farmhand.js

```diff
@@ -368,6 +368,7 @@ export default class Farmhand extends Component {
       sendPeerMetadata: null,
       selectedCowId: '',
       selectedItemId: '',
+      shovelUnlocked: false,
```

Comment:
Instead of adding a new state value, we can use `completedAchievements['gold-digger']` to determine whether the shovel is available similarly to how you did it here: https://github.com/jeremyckahn/farmhand/blob/cb4734a9727f2268b9cd3cbc9a1ac70a9d6d8510/src/utils.js#L1008-L1014

Though it's more verbose, I think it's a clean pattern and have used it elsewhere myself: https://github.com/jeremyckahn/farmhand/blob/cb4734a9727f2268b9cd3cbc9a1ac70a9d6d8510/src/components/Item/Item.js#L209-L214

Do you think it would make sense to drop this new state property and use the new `completedAchievements['gold-digger']` flag instead where necessary?

Reply:
will those persist? i.e. if the player takes out a new loan, will the game remember they had completed that achievement without re-evaluating the loan balance? i _think_ this achievement is a bit unique in how it unlocks

Reply:
They will, yes. `completedAchievements` properties are stored along with the rest of the persisted state object and don't change after being set to `true`. Additionally, achievements are only evaluated and updated when they have not yet been met: https://github.com/jeremyckahn/farmhand/blob/cb4734a9727f2268b9cd3cbc9a1ac70a9d6d8510/src/reducers.js#L1855-L1877

Reply:
ahh, great! that's perfect then. i like not adding to global state or save data 😄